utilities/sanitizeTrend.js

```diff
@@ -1,5 +1,13 @@
-const sanitizeTrend = (rawTimeline) => {
-  return JSON.parse(rawTimeline).default.timelineData.map((point) => {
+module.exports = (rawTimeline) => {
+  let parsedTimeline = {};
+
+  try {
+    parsedTimeline = JSON.parse(rawTimeline);
+  } catch (error) {
+    throw error;
```

Comment:
If we're expecting a JSON parse error, the most common sort of error is that the data has somehow made it to us in a pre-parsed format.  We may want to consider setting parsedTimeline to the rawTimeline in the catch block.